api/envoy/service/filter_chain/v3/fcds.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package envoy.service.filter_chain.v3;
+
+import "envoy/service/discovery/v3/discovery.proto";
+
+import "google/api/annotations.proto";
+
+import "envoy/annotations/resource.proto";
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.service.filter_chain.v3";
+option java_outer_classname = "FcdsProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/service/filter_chain/v3;filter_chainv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: FCDS]
+// [#not-implemented-hide:]
+// Filter Chain discovery :ref:`configuration overview <config_listeners>`
+
+service FilterChainDiscoveryService {
```

Comment:
As discussed in https://github.com/envoyproxy/envoy/pull/23096#discussion_r979046698 this is probably not needed.